web/app.js

```diff
@@ -988,7 +988,7 @@ const PDFViewerApplication = {
         if (!download) {
           return;
         }
-        PDFViewerApplication.download();
+        PDFViewerApplication.download(true /* isFallback */);
```

Comment:
Nit: In the code-base, this is *usually* formatted the following way.
```suggestion
        PDFViewerApplication.download(/* isFallback = */ true);
```